perception_dataset/rosbag2/rosbag2_to_non_annotated_t4_converter.py

```diff
@@ -460,10 +483,8 @@ def _convert_pointcloud(
             )
 
             # TODO(yukke42): Save data in the PCD file format, which allows flexible field configuration.
-            if isinstance(pointcloud_msg, PointCloud2):
-                points_arr = rosbag2_utils.pointcloud_msg_to_numpy(pointcloud_msg)
-            else:
-                points_arr = np.zeros((0, 5), dtype=np.float32)
```

Comment:
[Q]
Is there a specific reason for removing this? If not, could we keep it?

Reply:
Sorry I mistakenly left this modification. Fixed: https://github.com/tier4/tier4_perception_dataset/pull/58/commits/fa0acad29e13e1fbea9e0a5ae3d42567a465741c

Reply:
Instead of this, I am thinking that the checking instance here should be moved to before https://github.com/tier4/tier4_perception_dataset/blob/14d3e3170152d95787dd714e60601ae0c671f1be/perception_dataset/utils/rosbag2.py#L109-L112. 
What are your thoughts on this?

Reply:
Thank you! Fixed: https://github.com/tier4/tier4_perception_dataset/pull/58/commits/507473ca4d029d944d751d5c42723cbaf7974eed